topics/appendix/tools/intellij_platform_gradle_plugin/tools_intellij_platform_gradle_plugin.md

```diff
@@ -0,0 +1,290 @@
+
+| Name                          | Description                                                                                                                                       |
+|-------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------|
+| `releases`                    | IntelliJ Platform Releases repository.                                                                                                            |
+| `snapshots`                   | IntelliJ Platform Snapshots repository.                                                                                                           |
+| `nightly`                     | IntelliJ Platform Nightly repository, not available publicly.                                                                                     |
+| `ivy`                         | Local Ivy repository for resolving local Ivy XML files used for describing artifacts like local IntelliJ Platform instance, bundled plugins, etc. |
+| `marketplace`                 | JetBrains Marketplace Repository, hosts plugins for IntelliJ-based IDEs.                                                                          |
+| `jetbrainsRuntime`            | JetBrains Runtime (JBR), CloudFront-based hosting from which specific JBR releases are fetched, if explicitly requested.                          |
+| `binaryReleasesAndroidStudio` | Android Studio Binary Releases, required when running IntelliJ Plugin Verifier against Android Studio releases.                                   |
+| `binaryReleases`              | IntelliJ IDEA Binary Releases, required when running IntelliJ Plugin Verifier against JetBrains IntelliJ-based IDE releases.                      |
+| `recommended`                 | Applies a set of recommended repositories.                                                                                                        |
+
+#### Dependency Resolution Management
+
+{id="configuration.dependencyResolutionManagement"}
+
+To access the IntelliJ Platform Gradle Plugin within the <path>settings.gradle.kts</path> to use with `dependencyResolutionManagement`, add:
+
+```kotlin
+import org.jetbrains.intellij.platform.gradle.extensions.intellijPlatform
+
+plugins {
+    id("org.jetbrains.intellij.platform.settings") version "%intellij-platform-gradle-plugin-version%"
+}
+
+dependencyResolutionManagement {
+    repositoriesMode = RepositoriesMode.FAIL_ON_PROJECT_REPOS
+
+    repositories {
+        mavenCentral()
+    
+        intellijPlatform {
+            recommended()
+        }
+    }
+}
+```
+
+#### Cache Redirector
+
+{id="configuration.cacheRedirector"}
+
+Some repositories, by default, point to JetBrains Cache Redirector to provide better resources resolution.
```

Comment:
The "why would I need this" is missing from the description. If the JB Cache Redirector is better, why would I need to turn it off?